sphinxcontrib/openapi/openapi30.py

```diff
@@ -317,12 +323,32 @@ def openapihttpdomain(spec, **options):
             )
 
     # https://github.com/OAI/OpenAPI-Specification/blob/3.0.2/versions/3.0.0.md#paths-object
-    for endpoint in options.get('paths', spec['paths']):
-        for method, properties in spec['paths'][endpoint].items():
-            generators.append(_httpresource(
-                endpoint,
-                method,
-                properties,
-                render_examples='examples' in options))
+    if 'group' in options:
+        groups = collections.defaultdict(list)
+
+        for endpoint in options.get('paths', spec['paths']):
+            for method, properties in spec['paths'][endpoint].items():
+                key = properties.get('tags', [''])[0]
```

Comment:
>     * Can you please use `None` instead of `''`?

I can't because `None` and `''` are not comparable in Python 3 which breaks the sort I'm doing later. Open to other suggestions but this seemed easiest.

>     * Also, why do we use only the first tag from the list? I mean, shouldn't we this endpoint to other _groups_ as well?

I'm mimicking the behavior of editor.swagger.io, which just chooses the first one arbitrarily. I don't think we should be emitting documentation for the same API endpoint twice, and I don't see why we'd emit all tags for the header. Perhaps I could somehow include an index with all the tags after everything else but the usefulness seems minimal at best.